querydb/src/test/scala/io/joern/suites/JavaQueryTestSuite.scala

```diff
@@ -0,0 +1,48 @@
+package io.joern.suites
+
+import io.joern.console.scan._
+import io.joern.console.{Query, QueryBundle}
+import io.joern.javasrc2cpg.testfixtures.JavaSrcCode2CpgFixture
+import io.joern.util.QueryUtil
+import io.joern.x2cpg.testfixtures.TestCpg
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.{Call, Literal, Method, StoredNode}
+
+class JavaQueryTestSuite extends JavaSrcCode2CpgFixture(withOssDataflow = true) {
+  val argumentProvider = new QDBArgumentProvider(3)
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+  }
+
+  def queryBundle: QueryBundle = QueryUtil.EmptyBundle
+
+  def allQueries: List[Query] = QueryUtil.allQueries(queryBundle, argumentProvider)
+
+  def concatQueryCodeExamples: String =
+    allQueries
+      .map { q =>
+        q.codeExamples.positive
+          .mkString("\n")
+          .concat("\n")
+          .concat(
+            q.codeExamples.negative
+              .mkString("\n")
+          )
+      }
+      .mkString("\n")
+
+  def findMatchingCalls(cpg: Cpg, q: Query): List[String] = {
+    q(cpg).flatMap(_.evidence).collect { case c: Call => c.code }
```

Comment:
```suggestion
    q(cpg).flatMap(_.evidence).collectAll[Call].code.l
```

not sure if it's really better, just putting it out there to consider. 
same goes for the two queries below

Reply:
![Screenshot 2022-11-02 at 14 07 46](https://user-images.githubusercontent.com/28294550/199485998-eeff1607-f886-46b8-aa3c-6fd85f9952e8.png)
Unfortunately, we lose access to the property traversal steps. So I'll keep it as is for now.